CHANGELOG.md

```diff
@@ -1,19 +1,21 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
-## [Unreleased]
+## [1.0.0] - 2017-05-15
 - Record point-in-time CPU.
 - Multiple bug fixing on recaptool.
 - Multiple bug fixing on recap.
 - Refactor of recaplog.
 - Fix plesk mysql bug.
 - Install recaptool man page.
 - Adding specific requirements for bash(>=4) and sysstat(>=9).
+- Removing rpm spec requirements: elinks.
```

Comment:
See my other comment regarding RPM changelog versus the software's changelog.  This line should probably just be removed.